src/lit-element.ts

```diff
@@ -152,30 +172,33 @@ export class LitElement extends UpdatingElement {
   }
 
   /**
-   * Applies styling to the element shadowRoot using the `static get styles`
-   * property. Styling will apply using `shadowRoot.adoptedStyleSheets` where
+   * Applies styling to the element shadowRoot using the private styles
```

Comment:
I don't think we should mention a private field in a protected API doc

Reply:
I've reverted this comment. I think it's better the way it was.

My confusion came from... I think the tense is a bit ambiguous. By the time this function is called, we've already created the private "mapping" of styles (via `_getUniqueStyles()`), and this function doesn't call the `static get styles` method itself. Not a big deal, because we've _already_ called it, but that's why I was eager to modify the comment.